src/templates/render/failedFiles.html

```diff
@@ -1,5 +1,5 @@
 {% if data %}
-    <table id="{{ active_page }}" class="table table-sm text-start align-middle">
```

Comment:
This render also uses for queue page

Reply:
it seems to work well in both with the new id

Reply:
the problem with using the active_page is that the id changes when the web_root is set.

Reply:
still stays `failed` or `queue`, it's can't change because of web_root